mcs/class/monodoc/Resources/mdoc-html-utils.xsl

```diff
@@ -2336,27 +2336,20 @@
 				<xsl:if test="contains($membername, '&lt;')">
 					<xsl:value-of select="substring-before ($membername, '&lt;')" />
 				</xsl:if>
-				<xsl:text>``</xsl:text>
-				<xsl:value-of select="$numgenargs" />
+				<xsl:value-of select="'{'"/>
+					<xsl:for-each select="TypeParameters/TypeParameter">
+						<xsl:if test="not(position()=1)">, </xsl:if>
+						<xsl:value-of select="@Name"/>
```

Comment:
What happens when the method has the same name as an enclosing class? This is perfectly valid, though it generates a CS0693 warning:

	class C<T> {
		public static void M<T> ()
		{
		}
	}

So this would presumably generate a CREF of `M:C{T}.M{T}()`. Will that break anything?